### PR TITLE
Update ko translations

### PR DESCRIPTION
**Pull request by derornos**
- UI, log message
- base, prelude, promo cards

### CODE REVIEW

File: src/locales/ko/UI.json [hunk 1]
```diff
@@ -17,12 +17,342 @@
     "Blue": "파랑",
     "Black": "검정",
     "Purple": "보라",
+    "Your name": "당신의 이름",
+    "How to pay?": "비용을 어떻게 지불하시겠습니까?",
+    "The number of available actions on active cards": "",
+    "active": "플레이 시간",
+    "Select colonies to increase and decrease tile track": "트랙을 전진시키고 후퇴시킬 위성 타일들을 선택하세요.",
```
Comment (kberg) [inline src/locales/ko/UI.json:24]: These belong in colonies.json.

Comment (kberg) [reply src/locales/ko/UI.json:24]: ... I think.

File: src/locales/ko/moon_cards.json [hunk 2]
```diff
@@ -73,133 +73,133 @@
...
+  "Decrease your energy production 1 step and increase your plant production 1 step. Spend 1 titanium. Place a colony tile on the Moon and raise the Colony Rate 1 step.": "",
 
-  "Grand Luna Academy": "大月球学院",
-  "Draw 1 card per 2 Moon tags you have, including this.": "每2个月球标志抽一张牌，包括这张牌。",
+  "Grand Luna Academy": "",
+  "Draw 1 card per 2 Moon tags you have, including this.": "",
 
-  "HE3 Fusion Plant": "氦-3 核融合发电厂",
+  "HE3 Fusion Plant": "",
   "2 Mining rate": "",
```
Comment (kberg) [inline src/locales/ko/moon_cards.json:83]: A lot of these entries no long apply. This one is one example. You can take the entries in de/moon_cards.json as a reference.

File: src/locales/ko/UI.json (hunk 1 above)
Comment (derornos) [reply src/locales/ko/UI.json:24]: okay. i got it.

File: src/locales/ko/moon_cards.json [hunk 3]
```diff
@@ -73,133 +73,133 @@
   "Earth Embassy": "",
   "Effect: After being played, when you perform an action, your Moon tags count as Earth tags, but not vice versa.": "",
 
-  "Geodesic Tents": "测量营地",
```
[the remark this answers is not in the document]
Comment (derornos) [reply src/locales/ko/moon_cards.json:76]: it's cn, not ko.
: )

File: src/locales/ko/moon_cards.json (hunk 2 above)
Comment (derornos) [reply src/locales/ko/moon_cards.json:83]: I had removed the translation text after copying it from the Chinese file. However, missing items were discovered and dealt with this time.

Comment (kberg) [reply src/locales/ko/moon_cards.json:83]: What i mean is that `../de/moon_cards.json` is actually more accurate now. I went through and removed all the invalid entries. So if you took that one as its model, you'll have fewer unnecessary translations.

Comment (derornos) [reply src/locales/ko/moon_cards.json]: maybe... done?
: )

**Comment by kberg** [comment]
Sorry about the delay -- thank you!

**Review by kberg — APPROVED** [review]

**Comment by derornos** [comment]
you're welcome, thx : )

**Pull request merged**